apps/browser/src/autofill/services/autofill.service.ts

```diff
@@ -104,13 +106,26 @@ export default class AutofillService implements AutofillServiceInterface {
     frameId = 0,
     triggeringOnPageLoad = true,
   ): Promise<void> {
-    const mainAutofillScript = (await this.getOverlayVisibility())
+    // Autofill settings loaded from state can await the active account state indefinitely if
+    // not guarded by an active account check (e.g. the user is logged in)
+    const activeAccount = await firstValueFrom(this.accountService.activeAccount$);
+
+    // These settings are not available until the user logs in
+    let overlayVisibility = 0;
```

Comment:
You could use your named constant of this:
```suggestion
    let overlayVisibility = AutofillOverlayVisibility.Off;
```

(I don't think I see this imported into this context, so not sure a build would succeed if you accepted this)